aws-assume-role-policy/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "source_account_id" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>variable "source_account_id" is declared but not used